wp-includes/js/service-worker-navigation-routing.js

```diff
@@ -69,7 +69,11 @@ ERROR_OFFLINE_URL, ERROR_500_URL, NAVIGATION_DENYLIST_PATTERNS, ERROR_MESSAGES *
 
 							let body = text.replace(
 								'{{{WP_SERVICE_WORKER_ERROR_MESSAGE}}}',
-								errorMessages.error
+								'errorMessages.error'
```

Comment:
What is this line about? Why replace with `'errorMessages.error'`?

Reply:
Fixed 👍🏼